### PR TITLE
arraysAndCycles

### PR DESCRIPTION
**Pull request by AvetSargsyan**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+for (let i =0; i<movieNames.length; i++) {
+    console.log(movieNames[i]);
+};
+
+
+
+//2.Given an array of car manufacturers, convert the array to a string and back to an array
+let carManufac = ["Toyota", "Ford", "Honda"];
+console.log(carManufac.join().split());
+
```
Comment (artiombond) [inline script.js]: check please, how split method works
the result here [ 'Toyota,Ford,Honda' ] it's not the same array as before

File: script.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+//6.Filter array [1,6,7,8,3,4,5,6] by value > 3
+let arrOne = [1,6,7,8,3,4,5,6].filter((num)=>{
+    if (num > 3) return true
+});
+console.log(arrOne);
+
+
+//7.Write a function that takes two parameters - an array and a number and outputs the index of an array element equal to a number
+let arrTwo = [1,3,7,9,4,6,0];
+let num = 1;
```
Comment (artiombond) [inline script.js]: do we need this variable ?

File: script.js [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    if(num >1 ){
+        for(let i = 2; i<num ; i ++){
+            if(num % i == 0) {
+                prime = false;
+            };
+        };
+        if (prime) {
+            return num
+        }
+    };
```
Comment (artiombond) [inline script.js]: what if num < 1 ?

File: script.js [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+};
+let arrThree=[1,99,5,44,78,31,53,68,0,-6,2].filter(num=>{if(primeNums(num)) {
+    console.log(num);
+}});
```
Comment (artiombond) [inline script.js]: do not use filter for non-filter actions, you can use forEach here

File: script.js [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+
+
+//10.Implement a loop that prints odd numbers to the console
+function oddNums (num) {
+    if(num %2 !==0 && num !==2) {
+        return num
+    }
+};
+let arrFour=[1,2,3,99,5,0,44,-78,31,-53,68,1515252].filter(num=>{if(oddNums(num)) {
+    console.log(num);
```
Comment (artiombond) [inline script.js]: the same as in previous
mayby oddNums func should return true or false ?

Comment (AvetSargsyan) [reply script.js]: I have corrected all the mistakes in this file that you mentioned. Thanks . Can You check again ?

File: arraysAndCycles.js [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+      if (num % i == 0) {
+        prime = false;
+      }
+    }
+    if (prime) {
+      return true;
+    }
+  } else {
+    prime = false;
+  }
```
Comment (artiombond) [inline arraysAndCycles.js]: it's a bad practice, when function return value only in some cases, in the others we got undefined 
your function should return true or false in all cases
will it be enough to make only one return here? (return prime;)
and del condition 74 - 76

Comment (AvetSargsyan) [reply arraysAndCycles.js]: Is everything is right now ?

**Review by artiombond — APPROVED** [review]

**Pull request merged**